Libraries/libutil/Sources/DefaultProcessContext.cpp

```diff
@@ -112,6 +112,29 @@ executablePath() const
 #endif
 }
 
+static int commandLineArgumentCount = 0;
```

Comment:
Hmm... xcbuild doesn't have prefixes like that anywhere else, but also it's useful to know.